api/src/main/java/io/minio/MinioClient.java

```diff
@@ -1036,14 +1037,14 @@ private HttpResponse executeDelete(String bucketName, String objectName, Map<Str
    * @param data           HTTP request body data.
    */
   private HttpResponse executePost(String bucketName, String objectName, Map<String,String> headerMap,
-                                   Map<String,String> queryParamMap, Object data)
+                                   Map<String,String> queryParamMap, String contentType, Object data)
```

Comment:
The way it is written it is not possible we need to remove the MediaType behavior. Do you agree that is okay to do ?